app/assets/stylesheets/layout.scss

```diff
@@ -271,4 +283,16 @@ table tbody td[align="right"], table tbody th[align="right"], table thead th[ali
   &:visited {
     color: #4c2c92;
   }
+}
+
+.full-line {
+    border-bottom: 1px solid #bfc1c3;
```

Comment:
Not sure full-line is descriptive enough
I'd maybe split this into two classes `full-width` and `border-bottom`